src/Raven.Server/Web/System/AdminServerWideHandler.cs

```diff
@@ -87,89 +89,149 @@ public async Task PutServerWideBackupConfigurationCommand()
+        public Task GetServerWideTasksForStudio()
         {
             var taskName = GetStringQueryString("name", required: false);
 
             using (ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
             using (context.OpenReadTransaction())
             using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
             {
-                var backups = ServerStore.Cluster.GetServerWideBackupConfigurations(context, taskName);
-                ServerWideBackupConfigurationResults backupsResult = new ServerWideBackupConfigurationResults();
-                
-                foreach (var backupBlittable in backups)
+                var result = new ServerWideTaskConfigurations();
+
+                var blittables = ServerStore.Cluster.GetServerWideConfigurations(context, OngoingTaskType.Backup, taskName);
+                foreach (var blittable in blittables)
                 {
-                    var backup = JsonDeserializationServer.ServerWideBackupConfiguration(backupBlittable);
+                    var backup = JsonDeserializationServer.ServerWideBackupConfigurationForStudio(blittable);
                     backup.BackupDestinations = backup.GetFullBackupDestinations();
                     backup.IsEncrypted = backup.BackupEncryptionSettings != null &&
                                          backup.BackupEncryptionSettings.EncryptionMode != EncryptionMode.None;
+                    result.Backups.Add(backup);
+                }
 
-                    backupsResult.Results.Add(backup);
+                blittables = ServerStore.Cluster.GetServerWideConfigurations(context, OngoingTaskType.Replication, taskName);
+                foreach (var blittable in blittables)
+                {
+                    var externalReplication = JsonDeserializationCluster.ServerWideExternalReplication(blittable);
+                    result.ExternalReplications.Add(externalReplication);
                 }
-                
-                context.Write(writer, backupsResult.ToJson());
+
+                context.Write(writer, result.ToJson());
                 writer.Flush();
 
                 return Task.CompletedTask;
             }
         }
-        
-        // Toggle Disable/Enable for specified task
-        [RavenAction("/admin/configuration/server-wide/backup/state", "POST", AuthorizationStatus.ClusterAdmin)]
-        public async Task ToggleTaskState()
+
+        [RavenAction("/admin/configuration/server-wide/state", "POST", AuthorizationStatus.ClusterAdmin)]
+        public async Task ToggleServerWideTaskState()
         {
+            var typeAsString = GetStringQueryString("type", required: true);
+            var taskName = GetStringQueryString("name", required: true);
             var disable = GetBoolValueQueryString("disable") ?? true;
-            var taskName = GetStringQueryString("taskName", required: false);
```

Comment:
This is a bug, you must provide the name.
https://github.com/ravendb/ravendb/blob/564451a4742ba16ce44e45837d2b1df4112bd97e/src/Raven.Server/Web/System/AdminServerWideBackupHandler.cs#L158

It will just take the first task and disable/enable it